### DISCUSSION TITLE
Kyuubi frontend webUI status

### DISCUSSION THREAD

**Discussion by mmuru** [apache/kyuubi#2053] [Q&A] [answered]
Hi Team,
I noticed [issue-32](https://github.com/apache/incubator-kyuubi/issues/32) has been closed and wanted to follow up on its current status. When/which version of Kyuubi will start support this feature?  Thanks in advance.

**Comment by yaooqinn** [comment, marked as answer]
I tried to build it once https://github.com/apache/incubator-kyuubi/pull/1670, but I am not a good web developer.

**Reply by mmuru** [reply]
Thanks @yaooqinn. I will take a look at your PR.

**Reply by yaooqinn** [reply]
moved https://github.com/apache/incubator-kyuubi/pull/2061 for further discussion

**Reply by mmuru** [reply]
Great, I tested #2061 PR and looks good.
